gtpc/R3BGTPCProjector.cxx

```diff
@@ -11,11 +11,11 @@
 R3BGTPCProjector::R3BGTPCProjector()
     : FairTask("R3BGTPCProjector")
 {
-    fEIonization = 15.e-9;  // [GeV]
-    fDriftVelocity = 0.005; // [cm/ns]
-    fTransDiff = 0.0000001; // [cm^2/ns]
-    fLongDiff = 0.000001;   // [cm^2/ns]
-    fFanoFactor = 2;
```

Comment:
The fano factor for the gas should be one order less, Isn't it?

Reply:
Actually, the Fano factor should be less than 1 so that fluctuations are smaller than would be predicted based on Poisson statistics. The code is legacy from other previous one, so it could be a dummy value for testing. All parameters should be revised and moved to proper parameter containers a-la R3BRoot...